### PR TITLE
Add integration tests for find summary sample

### PR DESCRIPTION
**Pull request by daniel-verta**

### CODE REVIEW

File: client/verta/verta/monitoring/summaries/queries.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def __init__(
         self._labels = maybe(_labels_proto, labels)
         self._time_window_start = time_window_start
         self._time_window_end = time_window_end
-        self._aggregation = aggregation
+        self.aggregation = aggregation
```
Comment (convoliution) [inline client/verta/verta/monitoring/summaries/queries.py:218]: Is this shadowing the `@property` `aggregation` on line 247?

**Review by convoliution — CHANGES_REQUESTED** [review]

File: client/verta/verta/monitoring/summaries/queries.py (hunk 1 above)
Comment (daniel-verta) [reply client/verta/verta/monitoring/summaries/queries.py:218]: No, and in fact it should be causing the initializer to defer to the property setter. This way `self.aggregation = aggregation` invokes the validation logic baked into that setter.

Comment (daniel-verta) [reply client/verta/verta/monitoring/summaries/queries.py:218]:
```
In [1]: from verta.monitoring.summaries.queries import SummaryQuery, SummarySampleQuery
   ...:

In [2]: from verta._protos.public.monitoring.Summary_pb2 import (
   ...:     AggregationQuerySummary,
   ...:     FilterQuerySummarySample,
   ...:     FindSummaryRequest,
   ...:     FindSummarySampleRequest,
   ...:     LabelFilterQuerySummarySample,
   ...: )
   ...:

In [3]: empty_agg_proto = AggregationQuerySummary()

In [4]: q = SummarySampleQuery(aggregation=empty_agg_proto)

In [5]: q
Out[5]:
SummarySampleQuery(filter {
  find_summaries {
    page_number: 1
    page_limit: -1
  }
}
page_number: 1
page_limit: -1
)
```

Comment (convoliution) [reply client/verta/verta/monitoring/summaries/queries.py:218]: Ah yes I see it now!

**Review by convoliution — APPROVED** [review]

**Pull request merged**